databroker/core.py

```diff
@@ -442,6 +441,7 @@ def process(mds, fs, headers, func, fields=None, fill=False):
 
 def register_builtin_handlers(fs):
     "Register all the handlers built in to filestore."
+    from filestore import handlers, HandlerBase
```

Comment:
Yeah, I think that's right. Let's keep that in mind but wait until we are sure how the packaging in going to settle out overall.